spec/certificate_cleanup_spec.rb

```diff
@@ -0,0 +1,154 @@
+require 'spec_helper'
+require 'candlepin_scenarios'
+require 'pg'
+require 'mysql'
+
+describe 'Scheduled Jobs' do
```

Comment:
True. Writing them as integration tests would be better and I originally had them there but that's no longer doable since db specific queries won't work with hsqldb.

Reply:
Done. Rewrote the query so that the spec test is not needed.